terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -97,36 +97,44 @@ module "glue_demo_table" {
   glue_table_depends_on = [module.glue_database.db_name]
 }
 
-# Glue Cloud Platform ETL JOB
+# Glue Cloud Platform Ingestion Job (Load, Reload, CDC)
 module "glue_cloudplatform_reporting_job" {
   source                        = "./modules/glue_job"
   create_job                    = local.create_job
   name                          = "${local.project}-reporting-hub-${local.env}"
   description                   = local.description
+  command_type                  = "gluestreaming"
   create_security_configuration = local.create_sec_conf
-  job_language                  = "scala"
   temp_dir                      = "s3://${module.s3_glue_jobs_bucket[0].bucket.id}/tmp/reporting-hub/"
   checkpoint_dir                = "s3://${module.s3_glue_jobs_bucket[0].bucket.id}/checkpoint/reporting-hub/"
   spark_event_logs              = "s3://${module.s3_glue_jobs_bucket[0].bucket.id}/spark-logs/reporting-hub/"
   tags                          = local.all_tags
-  script_location               = "s3://${local.project}-artifact-store-${local.environment}/artifacts/cloud-platform/digital-prison-reporting-poc/cloud-platform-reporting-hub-vLatest.scala"
   enable_continuous_log_filter  = false
   project_id                    = local.project
   aws_kms_key                   = local.s3_kms_arn
   create_kinesis_ingester       = local.create_kinesis # If True, Kinesis Policies are applied
   additional_policies           = module.kinesis_stream_ingestor.kinesis_stream_iam_policy_admin_arn
-  timeout                       = 120
-  execution_class               = "FLEX"
+  timeout                       = 8
+  execution_class               = "STANDARD"
+  # Placeholder Script Location
+  script_location               = "s3://${local.project}-artifact-store-${local.environment}/artifacts/domain-platform/digital-prison-reporting-poc/place-holder-vLatest.scala"
```

Comment:
Should we get something into the jobs repo for this so you have a version controlled source?